libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -183,11 +227,6 @@ class CreatorNodeSelection extends ServiceSelection {
       }
     }
 
-    this.decisionTree.push({
```

Comment:
just curious why move this?

Reply:
just thought it made more sense at top level similar to all the other ones